pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-  "setuptools>=61.2",
+  "setuptools>=77.0",
```

Comment:
Let's see how the discussion on the other PR goes.

Reply:
As we already dropped py38 its most likely okay here